db/seeds.rb

```diff
@@ -24,7 +24,32 @@
 	description: "my description of Futurama",
 	small_cover_url: "/tmp/futurama.jpg",
 	large_cover_url: "/tmp/futurama.jpg",
-	category_id: 3)
+	category_id: 2)
```

Comment:
Hi Brandon,
Thanks for the tip.  I didn't think about clearing cookies.
--Sam


On Fri, Feb 20, 2015 at 11:13 AM, Brandon Conway <notifications@github.com>
wrote:

> In db/seeds.rb
> <https://github.com/SamSinger/myflix/pull/1#discussion_r25085044>:
>
> > @@ -24,7 +24,32 @@
> >  	description: "my description of Futurama",
> >  	small_cover_url: "/tmp/futurama.jpg",
> >  	large_cover_url: "/tmp/futurama.jpg",
> > -	category_id: 3)
> > +	category_id: 2)
>
> You should really seed a user, that will keep you from having to recreate
> your user every time and also will take care of this issue without having
> to clear your cookies. That is because a user with id 1 will exist.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SamSinger/myflix/pull/1/files#r25085044>.
>



-- 
Sam Singer